ethtypes/src/lib.rs

```diff
@@ -117,7 +120,8 @@ pub type Signature = H520;
 
 // Block Header
 
-#[derive(Debug, Clone, Serialize, Deserialize)]
+#[derive(Debug, Clone, BorshDeserialize)]
+#[cfg_attr(not(target_arch = "wasm32"), derive(Serialize, Deserialize))]
```

Comment:
Why `derive(Serialize, Deserialize)` is needed here and in other places?

Reply:
It's used in tests. A lot of test data files are in json

Reply:
I also noticed you moved related deps to dev-deps 👍